util/fibers/uring_socket.h

```diff
@@ -74,7 +74,28 @@ class UringSocket : public LinuxSocketBase {
     fd_ = (val << kFdShift) | (fd_ & ((1 << kFdShift) - 1));
   }
 
-  uint32_t error_cb_id_ = UINT32_MAX;
+  struct ErrorCbRefWrapper {
+    uint32_t error_cb_id = 0;
+    uint32_t ref_count = 2;
```

Comment:
Can you add a comment explaining why `2`? "1 for the callback and 1 for the member" or something like that? I hope I get it

Reply:
done